android/java/org/chromium/chrome/browser/crypto_wallet/util/AssetsPricesHelper.java

```diff
@@ -23,56 +18,8 @@ public static void fetchPrices(
             AssetRatioService assetRatioService,
             BlockchainToken[] assets,
             Callbacks.Callback1<HashMap<String, Double>> callback) {
+        // Method is not used - return empty HashMap to maintain compatibility
         HashMap<String, Double> assetsPrices = new HashMap<String, Double>();
```

Comment:
Let's remove completely if it is a no-op now

Reply:
Trying not to change too many files on Android in this PR. Here's an issue to clean up all the unused code, including `AssetPricesHelper.java`: https://github.com/brave/brave-browser/issues/37398